examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -480,11 +484,13 @@ class _AnimationDemoHomeState extends State<AnimationDemoHome> {
     }
   }
 
-  bool _handlePageNotification(ScrollNotification notification, PageController leader, PageController follower) {
+  bool _handlePageNotification(
+      ScrollNotification notification, PageController leader, PageController follower) {
     if (notification.depth == 0 && notification is ScrollUpdateNotification) {
       selectedIndex.value = leader.page;
-      if (follower.page != leader.page)
-        follower.position.jumpToWithoutSettling(leader.position.pixels); // ignore: deprecated_member_use
+      if (follower.page != leader.page) {
+        follower.position.jumpToWithoutSettling(leader.position.pixels);
+      } // ignore: deprecated_member_use
```

Comment:
moving that comment is probably a semantics-breaking change.

Reply:
disregard. looks like an issue with auto-switching to parens. my fault not allocating the comments correctly.